src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -667,11 +672,26 @@
 helps['network asg'] = """
     type: group
     short-summary: Manage application security groups.
+    long-summary: You can configure network security as a natural extension of an application’s structure, ASG allows
```

Comment:
Use YAML flows to break up very long summaries across multiple lines. The `>` flow is appropriate. See https://stackoverflow.com/a/21699210/2562556 for a fairly good description of `>` vs. `|` text flow operators.